CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to oah-bes-vm
+
+We welcome and appreciate contributions from the community to help make this project better. Your input and expertise are invaluable, and we're open to making the contribution process as smooth as possible. Please follow the guidelines below to contribute effectively.
+
+## First Time Setup
+You need an **Ubuntu-based** system to start contributing in oah-bes-vm.
+- ### Install prerequisites
+   * [Visual Studio Code](https://code.visualstudio.com/)
+   * [Vagrant](https://developer.hashicorp.com/vagrant/downloads/) 2.2.19 or higher
+   * [VirtualBox](https://www.virtualbox.org/wiki/Downloads) 7.0 or latest
+   * [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) 2.14.4 or latest
+   * [Git](https://git-scm.com/downloads)
+   * Vim editor: `sudo apt install vim`   
+- ### Install oah-shell
+      curl -L https://raw.githubusercontent.com/Be-secure/oah-installer/master/install.sh | bash
+- ### Update oah-init.sh 
+  - open `oah-init.sh` using vim editor.
+    ```bash
+    sudo vim $HOME/.oah/bin/oah-init.sh
```

Comment:
@pramit-d Is these lines necessary?

Exporting them from the terminal would be enough to change the namespace.
`export OAH_NAMESPACE=${OAH_NAMESPACE:=Be-Secure}` this line means that, if `OAH_NAMESPACE` is empty set it to `Be-Secure`.

So if we ask the user to export them manually, can't we avoid the steps where he edits the init file.

Moreover, theres a possibility that the user might push it along with the changes.

Reply:
@asa1997 Yes, we can skip these steps. User can export the namespace from the terminal without modifying the oah-init.sh file.

Reply:
@pramit-d Please do confirm the same.

Reply:
Done